### ISSUE TITLE
{{subtitles}} macro for post processing step passing content path instead of subtitle path

### ISSUE THREAD

**Issue by JonOfBallard** [morpheus65535/bazarr#2057]
**Describe the bug**
I'm using subcleaner as a post-processing step via Settings -> Subittles -> Command:
python3 /subcleaner/subcleaner.py "{{subtitles}}" -s

In my logs, I'm seeing this:
2023-02-07 12:23:44,692 - root                             (7f46b6490b38) :  INFO (post_processing:32) - BAZARR Post-processing result for file /data/media/movies/my movie/my movie.mp4 : Nothing returned from command execution

Note the .mp4 extension --- is the {{subtitles}} macro returning the wrong property? I don't see post_processing log lines for the actual subtitle files, so it doesn't appear that the right thing is happening also (making this extraneous.)

**To Reproduce**
1. clone subcleaner to your bazarr host
2. map subcleaner as a volume into bazarr docker container via compose
3. set post processing command as above

**Expected behavior**
For the subtitle file path to be passed to subcleaner via the {{subtitles}} for processing.

**Software (please complete the following information):**
Ubuntu 22.04.1 LTS

Docker
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Docker Buildx (Docker Inc., v0.9.1-docker)
  compose: Docker Compose (Docker Inc., v2.14.1)
  scan: Docker Scan (Docker Inc., v0.23.0)

Image
                "org.opencontainers.image.url": "https://github.com/linuxserver/docker-bazarr/packages",
                "org.opencontainers.image.vendor": "linuxserver.io",
                "org.opencontainers.image.version": "v1.1.4-ls181"

Thanks for looking

**Comment by morpheus65535** [comment]
This log line only state the media file for which a subtitles processing haven't happens properly. Your python script is either not returning any output or there's an issue with it. If you use `echo "{{subtitles}}"` you'll see the subtitles full file path in the log.

**Comment by morpheus65535** [comment]
In LSIO docker containers, app run as user abc, not as root.

**Comment by JonOfBallard** [comment]
Yep and I'm even running it as a specific user I'm providing, I was apparently thinking more about my nightly update script running git as root than the docker/fs permissions but I've sorted that. Interestingly after changing the command to echo and back I started getting a permission error from the script, so I'm not sure what was happening before that but I lost whatever state it was in. Thanks for the help!
